### PR TITLE
feat: arbitrum feature flag

### PR DESCRIPTION
**Pull request by alfetopito**
# Summary

Adds feature flag `isArbitrumOneEnabled`

# To Test

1. With the flag `on`, arb1 should be enabled and work both on CoW Swap and Explorer
2. With the flag `off`, arb1 shouldn't be visible nor work in either app

### CODE REVIEW

**Review by shoom3301 — APPROVED** [review]

**Review by elena-zh — COMMENTED** [review]
Hey @alfetopito , Explorer does not show the network even if the flag is enabled. 
Question: should we control widget behavior with this flag as well? Currently, it always shows the network when the flag is ON/OFF.

**Comment by alfetopito** [comment]
> Hey @alfetopito , Explorer does not show the network even if the flag is enabled. Question: should we control widget behavior with this flag as well? Currently, it always shows the network when the flag is ON/OFF. But when OFF, then I can see 'unsupported network' message, but I'd expect hiding the network from the selector. image

Thanks for checking!
I was waiting for the build to finish testing in the PR.
Will fix those issues.

File: libs/common-hooks/src/index.ts
Comment (alfetopito) [inline libs/common-hooks/src/index.ts:1]: Sorry about the noise.
Refactored common-hooks to extract hooks were here but are in fact cowswap only.
Also removed hooks that were not used.

File: libs/common-hooks/src/index.ts [hunk 1]
```diff
@@ -1,22 +1,17 @@
...
 export * from './useInterval'
 export * from './useIsOnline'
 export * from './useTheme'
-export * from './useToggle'
 export * from './useOnClickOutside'
 export * from './useFilterTokens'
 export * from './useTimeAgo'
 export * from './useIsWindowVisible'
-export * from './useGetReceipt'
-export * from './useBlockNumber'
```
Comment (alfetopito) [inline libs/common-hooks/src/index.ts:15]: Required wallet stuff, moved to coswap hooks

**Review by anxolin — APPROVED** [review]
Nice!

File: libs/common-hooks/src/useAvailableChains.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+import { SupportedChainId } from '@cowprotocol/cow-sdk'
+
+import { useFeatureFlags } from './index'
+
+export function useAvailableChains(): SupportedChainId[] {
+  const { isArbitrumOneEnabled } = useFeatureFlags()
+
+  return useMemo(
+    () => getAvailableChains(isArbitrumOneEnabled ? undefined : [SupportedChainId.ARBITRUM_ONE]),
+    [isArbitrumOneEnabled]
```
Comment (alfetopito) [inline libs/common-hooks/src/useAvailableChains.ts:13]: From now on, that's all we need to change to hide a `SupportedChainId` from the UI

**Review by elena-zh — APPROVED** [review]
Great, thank you!

**Pull request merged**